refinery/core/tests.py

```diff
@@ -2117,8 +2117,7 @@ def setUp(self):
             "auxiliary_nodes": [],
             "is_auxiliary_node": False,
             "file_extension": None,
-            "auxiliary_file_generation_task_state": None,
-            "ready_for_igv_detail_view": None
```

Comment:
Same in the tests: I think the idea behind `igv_detail_view` was that some FileTypes can be immediately visualized with IGV, but once you start zooming in things break due to the need of some type of index file being needed